### DISCUSSION TITLE
Is it possible to use two or more GPUs?

### DISCUSSION THREAD

**Discussion by LargeOgre** [neonbjb/tortoise-tts#329] [Q&A]
I get OOM error on my 6gig GPU but i got plenty more, is it possible to make use of a few of them? i currently have two hooked up but with nvidia-smi i see that only one is being utilized
I might be not understerstanding something fundamental about the technology , if so - sorry ;)

**Comment by eloop001** [comment]
I'm working on it. I have something working but I don't think it's very elegant.
